cmd/minishift/cmd/image/export.go

```diff
@@ -44,21 +44,21 @@ func exportImage(cmd *cobra.Command, args []string) {
 	defer api.Close()
 
 	if len(args) < 1 {
-		atexit.ExitWithMessage(0, "No image specifications provided")
+		atexit.ExitWithMessage(0, "You must specify a container image.")
```

Comment:
Again, you're not clarifying how to determine which images to export and how can users decide whether they want to export one, several, or all images.